tox.ini

```
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{36,37,38,39}-dj{22,31,32}
-    py310-dj{31,32}
-    py{38,39,310}-djmain
+    py{36,37,38,39}-dj{22,31}
+    py{36,37,38,39,310}-dj32
+    py{38,39,310}-dj{4.0,main}
```

I don't think this matches the below `dj40` dependency declaration with the `.`, it should be removed:

```suggestion
    py{38,39,310}-dj{40,main}
```

---

Yes, you are right. it was a typo!